Fastlane/deployment_lanes.rb

```diff
@@ -33,72 +33,75 @@
   if is_changed_since_last_tag == false
     tag_name = create_tag_name(xcodeproj: xcodeproj, target: target)
     slack_message(message: 'A new Release is cancelled as there are no changes since the last available tag.', tag_name: tag_name)
-  else
-    clear_derived_data
-    build_number = update_build_number(xcodeproj: xcodeproj, target: target)
-    tag_name = create_tag_name(xcodeproj: xcodeproj, target: target)
+    next
+  end
 
-    if options[:ci] || ENV['CI'] == 'true'
-      certs(app_identifier: options[:app_identifiers] || ENV['APP_IDENTIFIERS'])
-      prepare_for_ci
-    end
+  clear_derived_data
```

Comment:
I figured this would be a great change to have as well. It frustrates me that it's cleaning my derived data locally all the time when I'm debugging stuff 😅 

```suggestion
  if options[:ci] || ENV['CI'] == 'true'
          clear_derived_data
  end
```

Reply:
Ah, good point. That sounds like a pain to easily get rid of 👍